### PR TITLE
use grid in competition tab

### PR DESCRIPTION
**Pull request by ibeladi-mrsool**

### CODE REVIEW

**Comment by AifaDev** [comment]
Need more work

File: src/Components/Card.jsx [hunk 1]
```diff
@@ -4,14 +4,24 @@ export default function Card({ name, theme, startingDate, Deadline, status }) {
...
       <h4>{name}</h4>
-      <h5>Theme:</h5>
-      <p>{theme}</p>
-      <h5>Starting Date:</h5>
-      <p>{startingDate}</p>
-      <h5>Deadline:</h5>
-      <p>{Deadline}</p>
-      <h5>Status:</h5>
-      <p>{status}</p>
+      <div class="content">
```
Comment (mzaien) [inline src/Components/Card.jsx:7]: Does this work without className?

Comment (ibrahimBeladi) [reply src/Components/Card.jsx:7]: It is possible to make it work, but it will add dirty code instead
The use of classes is recommended

Comment (mzaien) [reply src/Components/Card.jsx:7]: I meant in jsx we use 'className' does the style apply with normal class?

Comment (ibrahimBeladi) [reply src/Components/Card.jsx:7]: It does work for me, I am not sure what are the differences between 'class' and 'className'
This is my first time working in jsx and react

**Pull request merged**